init.go

```diff
@@ -97,6 +75,14 @@ func initFlags(a *initFnCmd) []cli.Flag {
 	return append(fgs, routeFlags...)
 }
 
+func langsList() string {
+	allLangs := []string{}
+	for _, h := range langs.Helpers() {
+		allLangs = append(allLangs, h.LangStrings()...)
+	}
```

Comment:
given loop can be a bit simplified somehow in terms of the number of lists it creates

Reply:
example?